functions/operatingHours.ts

```diff
@@ -0,0 +1,90 @@
+import '@twilio-labs/serverless-runtime-types';
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import { responseWithCors, bindResolve, error500, success } from '@tech-matters/serverless-helpers';
+import moment from 'moment-timezone';
+
+const TokenValidator = require('twilio-flex-token-validator').functionValidator;
+
+type OperatingShift = { open: number; close: number };
+
+enum DaysOfTheWeek {
+  Monday = '1',
+  Tuesday = '2',
+  Wednesday = '3',
+  Thursday = '4',
+  Friday = '5',
+  Saturday = '6',
+  Sunday = '7',
+}
+
+type OperatingInfo = {
+  timezone: string; // the timezone the helpline uses
+  holidays: { [date: string]: string }; // a date (in MM/DD/YYYY format) - holiday name object to specify which days are holidays for the helpline
+  operatingHours: { [day in DaysOfTheWeek]: OperatingShift[] }; // object that pairs numbers representing weekdays to open and close shifts
+};
+
+type EnvVars = {
+  OPERATING_INFO_KEY: string;
+};
+
+export type Body = {};
+
+const isOpen = (hour: number) => (shift: OperatingShift): boolean =>
+  hour >= shift.open && hour < shift.close;
```

Comment:
This means we can't have opening hours that aren't on the hour, such as 8:30 - 11:30.  Would it be difficult to support that now?

Reply:
No it is not, my last commit adds this!